lib/agentx.c

```diff
@@ -332,7 +332,6 @@ static void smux_trap_multi_index_thd(struct event *thread)
 			return;
 		}
 		ringbuf_get(ibuf_ax, &notification_vars, sizeof(notification_vars));
-		zlog_err("%s get %p %p", __func__, notification_vars, *notification_vars);
```

Comment:
Could you amend this fix into f6efd0204b03af3ad9da558fba7c83426f5e5e3e?

Reply:
dine